tests/fun3d_tests/fully_coupled_disc_deriv_unsteady/turbulent-ATE/meshes/turbulent/Flow/fun3d.nml

```diff
@@ -0,0 +1,78 @@
+&project
+  project_rootname = 'miniMesh'
+/
+&raw_grid
+  grid_format = 'aflr3'
+  data_format = 'default'
+  swap_yz_axes = .false.
+/
+&governing_equations
+  eqn_type = 'compressible'
+  viscous_terms = 'turbulent'
+/
+&reference_physical_properties
+  mach_number     = 0.3
+  angle_of_attack = 5.0
+  reynolds_number = 2.91137e4
```

Comment:
Bump this Reynolds number back up to the value we were using previously for turbulence.